tests/test_data/Data_seperate_files_20170810_20180809/ISMN_qualityflags_description.txt

```diff
@@ -0,0 +1,51 @@
+ISMN Quality Flags

```

Comment:
i cannot see where this is used in the tests... therefore, it should not be included in the test data...

Reply:
not used at all, i deleted the file